src/eko/couplings.py

```diff
@@ -295,7 +291,9 @@ def couplings_expanded(order, couplings_ref, nf, scale_from, scale_to):
     lmu = np.log(scale_to / scale_from)
     res_as = expanded_qcd(couplings_ref[0], order[0], nf, lmu)
     res_aem = expanded_qed(couplings_ref[1], order[1], nf, lmu)
-    if order[0] >= 2 and order[1] >= 2:
+    # if order[0] >= 1 and order[1] >= 1:
+    # order[0] is always >=1
+    if order[1] >= 1:
```

Comment:
This if statement is different from the one before, can you remind why?

Reply:
In this case i only check whether order[1]>=1 since i'm adding the mixed terms in the evolution: for a_s we have it if order[0>=1 (always) and order[1]>=1 ; for a_em we have it if order[1] >=1 and order[0] >=1 (always)